app/views/plans/_navigation.html.erb

```diff
@@ -13,14 +13,17 @@
       </a>
     </li>
   <% end %>
-  <% if plan.editable_by?(current_user) %>
-    <li role="presentation" class="<%= (isActivePage(share_plan_path(plan)) ? 'active' : '') %>">
-      <a href="<%= share_plan_path(plan) %>" role="tab" aria-controls="content"><%= _('Share') %></a>
+  <% if plan.users.include?(current_user) %>
```

Comment:
this statement fixes the problem for now because plan#readable_by?(user_id) is doing more than it should, specially lines after https://github.com/DMPRoadmap/roadmap/blob/master/app/models/plan.rb#L379. We should remove can_super_admin? ... from there and put it as part of the policy methods for plan. If not, readable_by does not really reflect a readable role from the roles table.